tsl/test/sql/decompress_vector_qual.sql

```diff
@@ -0,0 +1,68 @@
+-- This file and its contents are licensed under the Timescale License.
```

Comment:
can we have explain output here currently it is not visible from test output whether vectorized filters are actually used here. also can we have some tests with NULL e.g.  `col < NULL`, `col <> NULL`, `col IS NULL`, ...

Reply:
* EXPLAIN would require versioned references, to avoid this I use the guc `debug_require_vector_qual` to check that the vectorized quals are used or not.
* `NullTest` is not vectorized currently, I added cases for `is null` and `is not null`.
* For null values in the column itself, I have tests with the `metric4` column.
* Not sure how to test `col < NULL` because all the functions we vectorize are strict, and we don't vectorize the initplan parameters yet.